### DISCUSSION TITLE
unknown option passwordrequirements

### DISCUSSION THREAD

**Discussion by santitoslp** [Ylianst/MeshCentral#4186] [Q&A] [answered]
I am trying to force 2fa for my users. I edited my config.json file with  "PasswordRequirements" option.
Restarted the server and when I go to "My Server" i get the message ADVERTENCIA: Opción de configuración desconocida "passwordrequirements" (unknown configuration option).
What am I doing wrong?

**Comment by santitoslp** [comment]
I was uing PasswordRequirements as level 1. When I moved it into domains section it worked

**Comment by si458** [comment, marked as answer]
https://github.com/Ylianst/MeshCentral/blob/aa58afcc3a5d738177ab7a7b6d0228d72af82b85/meshcentral-config-schema.json#L453-L480

PasswordRequirements should be `passwordRequirements`,
it should also be an object
it should also be placed inside you domains object

EXAMPLE:
```
"domains": {
  "": {
    "passwordRequirements": {
      "min": 10,
      "max": 20
    }
  }
}
```
